### ISSUE TITLE
strptime doesn't work with %z

### ISSUE THREAD

**Issue by joemarshall** [emscripten-core/emscripten#20467]
In musl, `strptime` and `struct tm` supports the %z option to read timezone into `tm.tm_gmtoff`. These don't work in emscripten.

**Comment by joemarshall** [comment]
damn, fixed in head.

**Comment by sbc100** [comment]
Could you share a failing example?  It think that would help in fixing the issue.

**Comment by RReverser** [comment]
@sbc100 FWIW this is linked to the https://github.com/emscripten-core/emscripten/pull/20470. I beleive it's ready for merge just waits for another round of review.

**Comment by sbc100** [comment]
#20470 lgtm, but its failing a bunch of tests tight now.

**Comment by RReverser** [comment]
Yeah but looks like mostly due to sizes that have changed in the interim. Rebase would probably fix it.
